src/datomic_rest_api/rest/gene.clj

```diff
@@ -82,7 +82,7 @@
   {:data
    (let [db   (d/entity-db gene)
          cds  (:gene/corresponding-cds gene)
-         data {:defined_by_mutation (not (empty? (:variation.gene/_gene gene)))
+         data {:defined_by_mutation (if (not (empty? (:variation.gene/_gene gene))) 1 0)
```

Comment:
Does the json not expect to be `true` or `false` rather than `1` or `0` ? (assume not, otherwise you'd not have coded it this way - it's just surprising!)